app/src/main/java/com/sun/noteapp/ui/textnote/TextNoteActivity.kt

```diff
@@ -61,25 +201,37 @@ class TextNoteActivity : AppCompatActivity(), View.OnClickListener {
         ).show()
     }
 
-    private fun showImageColorDialog() {
-        ImageColorDialog(
-            this,
-            R.layout.dialog_color,
-            object : BaseDialog.OnLoadDialogCallback<Int> {
-                override fun onSuccess(parrams: Int) {
-                    val mediumColor = ColorPicker.getMediumColor(parrams)
-                    val lightColor = ColorPicker.getLightColor(parrams)
-                    imageButtonHeaderColorTextNote.setBackgroundResource(mediumColor)
-                    toolbarTitle.setBackgroundResource(mediumColor)
-                    lineContentTextNote.setBackgroundResource(lightColor)
-                    horizontalScrollLabel.setBackgroundResource(mediumColor)
-                    buttonAlarmTextNote.setBackgroundResource(mediumColor)
-                    toolbarBottom.setBackgroundResource(mediumColor)
-                }
-            }).show()
+    private fun saveNote() {
+        val values = ContentValues()
+        values.put(NoteDatabase.NOTE_TITLE, editTitleTextNote.text.toString())
+        values.put(NoteDatabase.NOTE_CONTENT, lineContentTextNote.text.toString())
+        values.put(NoteDatabase.NOTE_TYPE, TYPE_TEXT_NOTE)
+        values.put(
+            NoteDatabase.NOTE_LABLE,
```

Comment:
```suggestion
            NoteDatabase.NOTE_LABEL,
```
chỗ này để làm gì nhỉ?